notification/method/webpush.php

```diff
@@ -140,10 +146,18 @@ public function notify()
 	{
 		$insert_buffer = new \phpbb\db\sql_insert_buffer($this->db, $this->notification_webpush_table);
 
+		// Load all users we want to notify
+		$notify_users = $this->load_receivers();
+
 		/** @var type_interface $notification */
 		foreach ($this->queue as $notification)
 		{
 			$data = $notification->get_insert_array();
+
+			// Choose receiving user's language
+			$user = $this->user_loader->get_user($data['user_id']);
```

Comment:
Or, should this be `$user = $this->user_loader->get_user($notification->user_id)` ?

Reply:
Infact that should be the same value. Shall I change it anyway?

Reply:
It's more consistent since we use `$this->user_loader->get_user($notification->user_id)` in other parts of this script

Reply:
Let's maybe rename this from `$user` so it is not ambiguous who the user is. Maybe something like:
`$recipient`
`$recipient_id`
`$user_to_notify`
`$notification_recipient`
etc..